www/shared/styles/imports/mixins/unstyled-button.css

```diff
@@ -0,0 +1,16 @@
+@define-mixin unstyled-button {
+    padding: 0;
+    border: none;
+    outline-style: none;
+    background: transparent;
+    border-radius: 0;
+    cursor: pointer;
+    appearance: none;
+
+    &:active,
+    &:focus,
+    &:visited,
+    &:hover {
+        outline: none;
```

Comment:
Unpopular opinion...

I would not remove the outline from the focus & active, since it hints keyboards users what is actually selected.

Or at least we should force a "button" component to have the active & focus state declared.

Reply:
I get your point but as this is an `unstyled-button`, I'd say it's expected for the dev to take care of things like that.

I see this more as of a kind of "button reset", but I can be wrong 🤷‍♂

Reply:
Removing it from the start makes it easier for the devs to forget totally about it. Just want to raise awareness for accessibility :)

Reply:
true, but focus & active need to be treated like a hover. If you are using this mixin, you need to have styles for all states.